integration/internal/network/network.go

```diff
@@ -0,0 +1,39 @@
+package network
+
+import (
+	"context"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/gotestyourself/gotestyourself/assert"
+)
+
+// TestNetworkConfig holds network configuration struct
+// used in api calls
+type TestNetworkConfig struct {
+	Name string
+	types.NetworkCreate
+}
+
+// Create creates a network with the specified options
+func Create(t *testing.T, ctx context.Context, client client.APIClient, expectError bool, ops ...func(*TestNetworkConfig)) string { // nolint: golint
```

Comment:
Not a huge fan on `boolean` arguments, especially on things like `expectError`. If we want to use this function in several cases (i.e. expect an error or not) then this function should just return `string, error` and we should let callers handle the error — we could even have a `CreateNoError` or `CreateT` (or whatever) that takes the `testing.T` pointer and expect no error.